app/partials/pet-details-form.php

```diff
@@ -138,7 +138,7 @@
         </p>
     </div>
 
-    <p class="h-spacing-base">
+    <p id="chip-number-wrap" class="h-spacing-base h-hide">
         <label for="chip-number">Chip number:</label>
```

Comment:
I'm wondering if there are any ARIA implications on the radio where we link what it's opening similar to how you would with accessibility for a dropdown. I'm not 100% sure though